### DISCUSSION TITLE
Understanding Code Structure

### DISCUSSION THREAD

**Discussion by GlauberF** [chatwoot/chatwoot#9325] [Contributors]
### Describe the bug

Hello Chatwoot team,

I'm interested in contributing to the project and would like to better understand the code structure to guide myself effectively. Could you provide information on how the code is organized, especially regarding the division between backend and frontend? Additionally, it would be helpful to know how the frontend is structured, for example, which folders correspond to which parts of the application.

### To Reproduce

1 - Division between Backend and Frontend:
How is the code divided between the backend and frontend?
Is there any specific architecture pattern being followed?
2 - Frontend Structure:
Which folders are specific to the frontend?
Could you provide an overview of which functionalities or components are contained in each frontend folder?

Thank you in advance for your assistance!

### Environment

app.chatwoot.com

### Cloud Provider

None

### Platform

None

**Comment by linear** [comment]
<p><a href="https://linear.app/chatwoot/issue/PR-1055/understanding-code-structure">PR-1055 Understanding Code Structure</a></p>

**Comment by yansena** [comment]
Hello i've been trying to contributing to the Chatwoot, I cloned and make some changes, but in many times I find me lost in the app structure, it was good find this issue and I'm commenting for follow them, because in documentation don't have a dev part that help developers to help with the app contribute
